pkg/riff/commands/doctor.go

```diff
@@ -86,21 +133,124 @@ func (*DoctorOptions) checkNamespaces(c *cli.Config) (bool, error) {
 	return ok, nil
 }
 
-func NewDoctorCommand(ctx context.Context, c *cli.Config) *cobra.Command {
-	opts := &DoctorOptions{}
+func (opts *DoctorOptions) getAccessCheckNamespace() string {
+	if opts.AllNamespaces {
+		return ""
+	} else {
+		return opts.Namespace
+	}
+}
 
-	cmd := &cobra.Command{
-		Use:     "doctor",
-		Aliases: []string{"doc"},
-		Short:   "check riff's requirements are installed",
-		Long: strings.TrimSpace(`
-    <todo>
-    `),
-		Example: "riff doctor",
-		Args:    cli.Args(),
-		PreRunE: cli.ValidateOptions(ctx, opts),
-		RunE:    cli.ExecOptions(ctx, c, opts),
+func convertSorted(checks []*ResourceAccessChecks) []*authv1.SelfSubjectAccessReview {
+	var result []*authv1.SelfSubjectAccessReview
+	for _, check := range checks {
+		for _, verb := range check.verbs {
+			result = append(result, &authv1.SelfSubjectAccessReview{
```

Comment:
I wonder if we should use a SelfSubjectRulesReview instead. It would involve a single request to the cluster to fetch the rules, but we'd need to compare the rules to the desired actions client side.

Reply:
I'd suggest to open an issue and do that in another PR.

Reply:
this issue tracks it: https://github.com/projectriff/cli/issues/31